src/utils/uemParser.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+export class UEMParser {
+    public static findSObjects(json: Object): Array<string> {
+        const sObjects = UEMParser.findObjectsWithValues(json, [
+            'mcf/list',
+            'mcf/timedList',
+            'mcf/genericLists'
```

Comment:
There is no 'mcf/genericLists' definition in MCF. The 'mcf/list' is basically for generic list.